config/configuration.toml.in

```diff
@@ -81,6 +81,22 @@ disable_block_device_use = @DEFDISABLEBLOCK@
 # behaving as it would when running on bare metal.
 # 
 #disable_nesting_checks = true
+#
+# Internetworking model
+# Determines how the VM should be connected to the
+# the container network interface
+# Options:
+#
+# - bridged
```

Comment:
I'd be tempted to indent these two options (and indent their descriptions further) for clarity:

```
# Options:
#
#   - bridged
#
#     Uses a linux bridge to interconnect
#     the container interface to the VM. ...
#          :
#
#   - macvtap
#
#     Used when the Container network
#          :
#

```